packages/example-rpc-server/tslint.build.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
We don't need to a tslint file when using `lb-tslint`. This file can be removed.